Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ruby:3.0.4-alpine3.16
```

Comment:
You could improve the Dockerfile by going through the recommendations from best practices like this: https://lipanski.com/posts/dockerfile-ruby-best-practices